### PR TITLE
issue/3199 Added _shouldExpandFirstItem and converted to ES6 import/export

### PR DESCRIPTION
**Pull request by oliverfoster**
fixes https://github.com/adaptlearning/adapt_framework/issues/3199

### Added
* `_shouldExpandFirstItem`

### Changed
* Converted to ES6 import/export from requirejs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +42,8 @@ guide the learner’s interaction with the component.
 
 **\_shouldCollapseItems** (boolean): Used to control the auto-collapse of other expanded Accordion items whenever an item is clicked/expanded. Defaulted to 'true'
 
+**\_shouldExpandFirstItem** (boolean): Used to control the if the first item will expanded by default. Defaulted to 'false'
```
Comment (cahirodoherty-learningpool) [inline README.md:45]: Minor: Used to control ~the~ if the first item

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Review by kirsty-hames — APPROVED** [review]
great thanks - works as expected

**Comment by kirsty-hames** [comment]
I approved before looking at the accessibility sorry @oliverfoster. I've just noticed that the `aria-expanded` is `false` when enabling `_shouldExpandFirstItem`. Once you interact with the component it updates to the relevant expanded state but this should be set to `true` initially.

File: templates/accordion.hbs [hunk 2]
```diff
@@ -7,7 +7,7 @@
     {{#each _items}}
     <div class="accordion__item{{#if _graphic.src}} has-image{{/if}}{{#if _classes}} {{_classes}}{{/if}}" data-index="{{_index}}">
 
-      <button id="{{../_id}}-{{@index}}-accordion-button" class="accordion__item-btn js-toggle-item is-closed{{#if _isVisited}} is-visited{{/if}}" aria-expanded="false">
+      <button id="{{../_id}}-{{@index}}-accordion-button" class="accordion__item-btn js-toggle-item{{#if _isActive}} is-selected is-open{{else}} is-closed{{/if}}{{#if _isVisited}} is-visited{{/if}}" aria-expanded="false">
```
Comment (oliverfoster) [inline templates/accordion.hbs:10]:
```suggestion
      <button id="{{../_id}}-{{@index}}-accordion-button" class="accordion__item-btn js-toggle-item{{#if _isActive}} is-selected is-open{{else}} is-closed{{/if}}{{#if _isVisited}} is-visited{{/if}}" aria-expanded="{{#if _isActive}}true{{else}}false{{/if}}">
```

**Review by kirsty-hames — APPROVED** [review]

**Review by eleanor-heath — APPROVED** [review]

**Pull request merged**